src/Elastic.Apm.AspNetCore/Extensions/TransactionExtentions.cs

```diff
@@ -13,26 +14,34 @@ internal static class TransactionExtensions
 		/// </summary>
 		/// <param name="transaction">Transaction object</param>
 		/// <param name="isForError">Is request body being captured for error (otherwise it's for transaction)</param>
-		/// <param name="request">Current http request</param>
+		/// <param name="httpRequest">Current http context containing the http request</param>
```

Comment:
It sounds quite confusing - `httpRequest`'s type is `HttpRequest ` so why is it `http context`?

Reply:
That's merge leftover, fixed it - thanks.